src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+:root {
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo__img {
+  display: flex;
+  height: 40px;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  margin: 0;
+  padding-left: 0;
+}
+
+.nav__item {
+  display: flex;
+  text-align: center;
+}
+
+.nav-link {
+  color: #000;
+  text-decoration: none;
+  position: relative;
+  font-weight: 500;
+  font-size: 12px;
+  height: 60px;
+  line-height: 60px;
+  align-items: center;
+  text-transform: uppercase;
+  margin-left: 20px;
+}
```

Comment:
The class name '.nav-link' should be consistent with the BEM naming convention used elsewhere in your CSS. It should be '.nav__link' to indicate that it's an element of the 'nav' block.